setup.cfg

```diff
@@ -0,0 +1,210 @@
+[flake8]
+exclude =
```

Comment:
Should we also consider adding an "ignore" list for errors and warnings that we don't want to care about yet?

For what it's worth, Traits took that approach: not excluding individual files, but ignoring warnings and errors and then slowly reducing the number of ignored error codes over time. Some combination of the two is probably the right approach.

Reply:
Good point.
I think a `per-file-ignores = */api.py:F401` does make sense here.

But I won't, for example, ignore `E722 do not use bare 'except'` because I think we do care about it and in order to remove it from the ignore will require changes across many files, making it harder to incrementally lint the entire code base.